### PR TITLE
Edited code blocks

### PR DESCRIPTION
**Pull request by yeremee**
# Edited code blocks

The code snippets alternated between them. 
Some pieces of code were not displayed.

Snippet* tags is deleted from cs/vb files.

### CODE REVIEW

File: samples/snippets/csharp/VS_Snippets_Wpf/WindowClosingSnippets/CSharp/DataWindow.xaml.cs [hunk 1]
```diff
@@ -1,4 +1,3 @@
-//<SnippetWindowClosingCODEBEHIND1>
 using System; // EventArgs
 using System.ComponentModel; // CancelEventArgs
 using System.Windows; // window
```
Comment (svick) [inline samples/snippets/csharp/VS_Snippets_Wpf/WindowClosingSnippets/CSharp/DataWindow.xaml.cs]: Are comments indicating why a `using` is needed actually useful? (Though this applies to more than just this snippet, so maybe that should be a separate PR.)

Also, why is the namespace here `CSharp` (and `VisualBasic`), when all the other snippets use `SDKSample`?

Comment (mairaw) [inline samples/snippets/csharp/VS_Snippets_Wpf/WindowClosingSnippets/CSharp/DataWindow.xaml.cs]: @svick please open an issue for us to look at this. since we have a bunch of legacy code snippets, we'll find all sorts of "treasures" in there 😄

**Comment by mairaw** [comment]
@yeremee before we can look into this, we'd need you to sign the CLA. Please check your inbox.

File: samples/snippets/csharp/VS_Snippets_Wpf/WindowClosingSnippets/CSharp/DataWindow.xaml.cs (hunk 1 above)
Comment (svick) [inline samples/snippets/csharp/VS_Snippets_Wpf/WindowClosingSnippets/CSharp/DataWindow.xaml.cs]: @mairaw Here's a PR removing the comments: https://github.com/dotnet/docs/pull/2060.

**Comment by yeremee** [comment]
@mairaw CLA is signed

**Comment by mairaw** [comment]
@yeremee ok, thanks. @jongalloway the PR hasn't updated the CLA label. can you help?

**Comment by mairaw** [comment]
@yeremee do you wanna try going to https://cla2.dotnetfoundation.org/ and doing the process again? We had someone from our team with a similar issue and now his PR shows up as signed.

**Comment by yeremee** [comment]
@mairaw Done. `cla-required` replaced by `cla-signed`.

**Review by rpetrusha — APPROVED** [review]
Thanks for correcting these code blocks, @yeremee!
